docs/create/azure/terraform-private-endpoint/index-terraform-private-endpoint.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: index-azure-terraform-private-endpoint
+title: Azure Cache for Redis Enterprise using Terraform with Private Link
+sidebar_label: Private Link Terraform
+slug: /create/azure/terraform-private-endpoint
```

Comment:
Sulg does not appear in the sidebar, will need to be added to `sidebar.js`

Reply:
I haven't finished moving those resources, so I just left it out of the sidebar.  Now I realize I should complete moving those resources because its searchable.

Reply:
@dashaun Would love to add sidebar once the tutorial is completed.